test/project_types/script/layers/application/push_script_test.rb

```diff
@@ -50,18 +51,23 @@
     subject { Script::Layers::Application::PushScript.call(ctx: @context, force: force) }
 
     it "should prepare and push script" do
-      script_service_instance = Script::Layers::Infrastructure::ScriptService.new(ctx: @context)
+      script_service_instance = Script::Layers::Infrastructure::ScriptService.new(ctx: @context, api_key: api_key)
```

Comment:
🤔 why don't we just return mock here?

Reply:
No idea. I avoided changing the tests as much as possible, keeping them as status quo, as the goal of this PR was not to address any weirdness/issues within them. I think it's generally a better idea to avoid changing the tests too much when refactoring, if possible, so that you can test if the refactors broken anything in the tests.

I think these changes will let us clean the tests up, though.